### PR TITLE
added Bitrise to Integrations

### PR DESCRIPTION
**Pull request by BanyikAnna**
+ rearranged CI Integrations list items alphabetically. Pls have a look!

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -283,14 +283,15 @@ pages:
...
-    - CircleCI Integration: best-practices/continuous-integration/circle-ci.md
-    - Travis Integration: best-practices/continuous-integration/travis.md
-    - Bamboo Integration: best-practices/continuous-integration/bamboo.md
-    - GitLab CI Integration: best-practices/continuous-integration/gitlab.md
-    - Azure DevOps Integration: best-practices/continuous-integration/azure-devops.md
-    - Nevercode Integration: best-practices/continuous-integration/nevercode.md
+    - Azure DevOps: best-practices/continuous-integration/azure-devops.md
+    - Bamboo: best-practices/continuous-integration/bamboo.md
+    - Bitrise: best-practices/continuous-integration/bitrise.md
+    - CircleCI: best-practices/continuous-integration/circle-ci.md
```
Comment (janpio) [inline mkdocs.yml]:
```suggestion
    - CircleCI: best-practices/continuous-integration/circle-ci.md
```

File: mkdocs.yml [hunk 2]
```diff
@@ -283,14 +283,15 @@ pages:
...
-    - GitLab CI Integration: best-practices/continuous-integration/gitlab.md
-    - Azure DevOps Integration: best-practices/continuous-integration/azure-devops.md
-    - Nevercode Integration: best-practices/continuous-integration/nevercode.md
+    - Azure DevOps: best-practices/continuous-integration/azure-devops.md
+    - Bamboo: best-practices/continuous-integration/bamboo.md
+    - Bitrise: best-practices/continuous-integration/bitrise.md
+    - CircleCI: best-practices/continuous-integration/circle-ci.md	
+    - GitLab CI: best-practices/continuous-integration/gitlab.md
+    - Jenkins: best-practices/continuous-integration/jenkins.md
+    - NeverCode: best-practices/continuous-integration/nevercode.md
```
Comment (janpio) [inline mkdocs.yml]:
```suggestion
    - NeverCode: best-practices/continuous-integration/nevercode.md
```

**Comment by BanyikAnna** [comment]
Hi @janpio still 404 when clicking on Bitrise [here](https://docs.fastlane.tools/best-practices/continuous-integration/bitrise/). I'm wondering what else I can modify 🤔 🙂 I appreciate any suggestion 🚀

**Comment by janpio** [comment]
That is the normal website - there it will only be deployed after I merge the PR. But here is the preview where everything is working: https://deploy-preview-917--fastlane-docs-preview.netlify.com/best-practices/continuous-integration/bitrise/

**Comment by BanyikAnna** [comment]
Thanks @janpio for all your help! 🙏 😊

**Pull request merged**